Policies/Kubernetes/allowed-seccomp-profiles/rego/src_test.rego

```diff
@@ -0,0 +1,388 @@
+package k8sazureallowedseccomp
+
+test_input_seccomp_empty_parameters {
+    input := { "review": input_review_pod_single, "parameters": input_parameters_empty}
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_input_seccomp_allowed_all {
+    input := { "review": input_review_pod_single, "parameters": input_parameters_wildcard}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seccomp_allowed_in_list {
+    input := { "review": input_review_pod_single, "parameters": input_parameter_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seccomp_not_allowed_not_in_list {
+    input := { "review": input_review_pod_single, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_input_seccomp_pod_multiple_empty_parameters {
+    input := { "review": input_review_pod_multiple, "parameters": input_parameters_empty}
+    results := violation with input as input
+    count(results) == 2
+}
+
+test_input_seccomp_pod_multiple_allowed_all {
+    input := { "review": input_review_pod_multiple, "parameters": input_parameters_wildcard}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seccomp_pod_multiple_allowed_in_list {
+    input := { "review": input_review_pod_multiple, "parameters": input_parameter_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seccomp_pod_multiple_not_allowed_not_in_list {
+    input := { "review": input_review_pod_multiple, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 2
+}
+
+test_input_seccomp_not_allowed_no_annotation {
+    input := { "review": input_review_no_annotation, "parameters": input_parameter_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_input_seccomp_not_allowed_multiple_no_annotation {
+    input := { "review": input_review_containers_no_annotation, "parameters": input_parameter_in_list}
+    results := violation with input as input
+    count(results) == 2
+}
+
+test_input_seccomp_container_allowed_all {
+    input := { "review": input_review_container, "parameters": input_parameters_wildcard}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seccomp_container_allowed_in_list {
+    input := { "review": input_review_container, "parameters": input_parameter_in_list}
```

Comment:
Can you add another test for input_review_container and input_parameters_in_list?

Reply:
What do you want the second test to test?

[Line 87](https://github.com/Azure/Community-Policy/pull/64/files/e2351bb95c409da9800ccc694c0dc14b834e375a#diff-96bfc96ab40793098c2efdd40f95e76fR87) has a test for multiple containers and in list.

Reply:
Line 87 test is multiple containers are present and there is a single item in parameter array.
My comment is to test a scenario where there is a single container and multiple items in input parameter array.

Good to have extensive coverage in unit tests as it helps in future if someone edits the policy and by mistake introduces a regression.